### PR TITLE
fix: Sync debian packaging with Ubuntu

### LINKED ISSUE #53: nvidia powermanagement files missing
I'm trying to follow the recommendations from nvidia [`powermanagement` page](https://download.nvidia.com/XFree86/Linux-x86_64/450.57/README/powermanagement.html) to solve the issue of rendering corruption after suspend or hibernate.

The `systemd` service files present in [ubuntu version of `nvidia-driver-440`](https://packages.ubuntu.com/focal/amd64/nvidia-driver-440/filelist) are missing in the [pop version](https://launchpad.net/~system76/+archive/ubuntu/pop/+sourcepub/11403967/+listing-archive-extra).

Is there a reason for that?

My laptop usually fails after recovering from a suspend or hibernate, see [here](https://unix.stackexchange.com/questions/600805/fuzzy-graphics-after-suspend-win-nvidia-drivers/600857#600857)

### PR DESCRIPTION
**Pull request by mmstick**
Ubuntu's made a lot of changes to their Debian packaging, and this will include some files that were previously missing in our packaging.

Fixes #53

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -81,7 +88,9 @@ Replaces:
 Provides:
  nvidia-dkms-kernel
 Depends:
- dkms, nvidia-kernel-source-440, nvidia-kernel-common-440,
+ dkms, nvidia-kernel-source-440,
+ nvidia-kernel-common-440 (<= 440.100-1),
```
Comment (jacobgkau) [inline debian/control:92]: Still having problems upgrading after adding this branch:

```
system76@pop-os:~$ sudo apt upgrade
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Calculating upgrade... Done
The following packages have been kept back:
  libnvidia-cfg1-440 libnvidia-compute-440 libnvidia-compute-440:i386 libnvidia-decode-440
  libnvidia-decode-440:i386 libnvidia-encode-440 libnvidia-encode-440:i386 libnvidia-fbc1-440
  libnvidia-fbc1-440:i386 libnvidia-gl-440 libnvidia-gl-440:i386 libnvidia-ifr1-440
  libnvidia-ifr1-440:i386 nvidia-compute-utils-440 nvidia-kernel-source-440 nvidia-utils-440
  xserver-xorg-video-nvidia-440
The following packages will be upgraded:
  libnvidia-common-440 nvidia-kernel-common-440
2 upgraded, 0 newly installed, 0 to remove and 17 not upgraded.
```

Attempting to force any of the kept-back packages to upgrade results in `nvidia-driver-440` and `system76-driver-nvidia` being removed. The Pop!_Shop is showing 19 Operating System Updates available, but the "Update" button doesn't install the updates.

This line seems to be the problem: `nvidia-driver-440 : Depends: nvidia-kernel-common-440 (<= 440.100-1) but 440.100-1pop2~1596222574~20.04~fcb286b is to be installed`

Comment (jacobgkau) [reply debian/control:92]: It seems a little strange to depend on `>= 440.100` and `<= 440.100-1` of the same package.

Comment (mmstick) [reply debian/control:92]: I thought for sure that I removed all those version checks. Let me do another check

**Comment by jacobgkau** [comment]
Tested with:
- GTX 1650 in gaze14
- RTX 2070 in addw1
- RTX 2080 Super in thelio-b1.6

External display with audio output, CUDA processing, video playback in Firefox, and mode switching all work. Display functions normally when booting with or without external display connected.

I'm noticing that `nvidia-driver-440-server` is offered in the Pop!_Shop starting the next reboot after `libnvidia-extra-440` is installed. This is a new package that is introduced by this update. @mmstick Is that something you can fix in `libnvidia-extra-440`, or should I open an issue in the Pop!_Shop repo?

**Comment by mmstick** [comment]
That's strange. This would be an issue in Pop!_Shop.

**Pull request merged**